sdk/tables/Azure.Data.Tables/src/Extensions/DictionaryTableExtensions.cs

```diff
@@ -64,6 +64,10 @@ internal static class DictionaryTableExtensions
                 }
             }
 
+            // Remove the Timestamp property, as it is controlled by the service not the client.
```

Comment:
I think it might be better to just add an extra condition to line 33

```c#
if (item.Key == TableConstants.PropertyNames.ETag || item.Key == TableConstants.PropertyNames.Timestamp)
```

This avoids some extra work to generate the annotation property name that we just end up having to remove.

Reply:
Done. Also did this on the other extension method.